### ISSUE TITLE
Active facet header should indicate values selected to screen readers

### ISSUE THREAD

**Issue by jcoyne** [projectblacklight/blacklight#3241]
Currently we indicate a facet has something selected by using only the background color.

<img width="404" alt="Screenshot 2024-07-30 at 9 27 07 AM" src="https://github.com/user-attachments/assets/911192e4-3745-40ed-9c61-fc8a27dbdf6b">

We should consider a design like this that indicates the number of selected values with text:
<img width="259" alt="Screenshot 2024-07-30 at 9 28 36 AM" src="https://github.com/user-attachments/assets/48791682-4765-4375-b697-56b498e014a2">
